clarified authz_header value (#263)
openid/sharedsignals: openid-sharedsignals-framework-1_0.md

authorization_header

> > If the endpoint_url requires authorization, the receiver SHOULD provide this authorization header in the stream creation/updation. If present, the Transmitter MUST provide this value with every HTTP request to the `endpoint_url`. The Transmitter MUST specify the value of this field as the value of the `Authorization` HTTP Request Header as specified in Section 11.6.2 of RFC 9110 {{RFC9110}}

Reviewer: updation -> update
combined last two sentences

```suggestion
> If the endpoint_url requires authorization, the receiver SHOULD provide this authorization header in the stream creation/update. If present, the Transmitter MUST include this value in every HTTP request to the endpoint_url, specifying it as the `Authorization` HTTP Request Header as detailed in Section 11.6.2 of RFC 9110  {{RFC9110}}
```
